db/create_tables.sql

```diff
@@ -55,6 +55,8 @@ CREATE TABLE IF NOT EXISTS `state_machines` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `name` VARCHAR(255) NOT NULL,
   `version` TINYINT UNSIGNED NOT NULL,
+  `start_state_id` BIGINT(20) NOT NULL,
```

Comment:
We can ignore this if we are generating a DAG for SM, and start execution of all independent states.

The state machine class has start state variable, so thought we are going with it.